uint8 num_channels
---
bool result
